articles/quantum-concepts-4-Qubit.md

```diff
@@ -109,7 +109,9 @@ While the previous constitute the most popular primitive gates for describing op
 The simplest such primitive is the single qubit-rotation. Three single-qubit rotations are typically considered: $R_x$, $R_y$ and $R_z$. To visualize the action of the rotation $R_x(\theta)$, for example, imagine pointing your right thumb along the direction of the $x$-axis of the Bloch sphere and rotating the vector with your hand through an angle of $\theta/2$ radians. This confusing factor of $2$ arises from the fact that orthogonal vectors are $180^\circ$ apart when plotted on the Bloch sphere, yet are actually $90^\circ$ degrees apart geometrically. The corresponding unitary matrices are:
 
 $$
-R_z(\theta) = \begin{bmatrix} e^{-i\theta/2} & 0\\\\  0& e^{i\theta/2} \end{bmatrix},\qquad R_x(\theta) = H R_z(\theta) H, \qquad R_y(\theta) = SHR_z(\theta)HS^\dagger.
+R_z(\theta) = e^{-i\theta Z/2} = \begin{bmatrix} e^{-i\theta/2} & 0\\  0& e^{i\theta/2} \end{bmatrix}, \\
```

Comment:
Unfortunately, due to some bugs on our end, some LaTeX notation doesn't work without modification. In particular, `\` is special to Markdown, and so needs to escaped as `\\`, meaning that linebreaks are `\\\\`. We hope to solve these issues to make our LaTeX notation better agree with conventional notation, but in the meantime, would you be willing to make changes similar to the suggested edits below?

```suggestion
R_z(\theta) = e^{-i\theta Z/2} = \begin{bmatrix} e^{-i\theta/2} & 0\\\\  0& e^{i\theta/2} \end{bmatrix}, \\\\
```

Reply:
Thanks for letting me know about this! I did see that but didn't realize it was necessary for your documentation. I'll make sure to modify it.